x/qgb/config.go

```diff
@@ -0,0 +1,28 @@
+package qgb
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	// bech32PrefixAccAddr defines the bech32 prefix of an account's address
+	bech32PrefixAccAddr = "qgb"
+	// bech32PrefixAccPub defines the bech32 prefix of an account's public key
+	bech32PrefixAccPub = "qgbpub"
+	// bech32PrefixValAddr defines the bech32 prefix of a validator's operator address
+	bech32PrefixValAddr = "qgbvaloper"
+	// bech32PrefixValPub defines the bech32 prefix of a validator's operator public key
+	bech32PrefixValPub = "qgbvaloperpub"
+	// bech32PrefixConsAddr defines the bech32 prefix of a consensus node address
+	bech32PrefixConsAddr = "qgbvalcons"
+	// bech32PrefixConsPub defines the bech32 prefix of a consensus node public key
+	bech32PrefixConsPub = "qgbvalconspub"
+)
+
+func init() {
+	config := sdk.GetConfig()
+	config.SetBech32PrefixForAccount(bech32PrefixAccAddr, bech32PrefixAccPub)
+	config.SetBech32PrefixForValidator(bech32PrefixValAddr, bech32PrefixValPub)
+	config.SetBech32PrefixForConsensusNode(bech32PrefixConsAddr, bech32PrefixConsPub)
```

Comment:
while it might not be done using this exact mechanism, this is already done using `celes`, so we should avoid doing this again using `qgb`. Surprised this doesn't break anything lol

Reply:
This is defined inside the module... maybe it is picked up per module?

Reply:
It makes sense for them to have it in their module, as they want to have grav addresses, but not ours as we will have celes. sdk.GetConfig is global (hence the "sealing"), and having different address types per module is not an sdk feature I'm aware of. The only addresses that we should have to manage are done so via the orchestrator and eth addresses, which we are keeping in the state. 

Do things break when we remove them?

Reply:
not at all. Ill remove them then.